dashboard/db/migrate/20160315195700_associate_answers_with_modules_not_tasks.rb

```diff
@@ -0,0 +1,9 @@
+class AssociateAnswersWithModulesNotTasks < ActiveRecord::Migration
+  def change
+    #At the time this migration was written there were no Evaluation Answers in production, so it's safe to blow
+    #everything up
+    Plc::EvaluationAnswer.destroy_all
```

Comment:
The up migration, it's not necessary. You can simply say "Associate every evaluation answer with the module that it's task was associating with."

The down migration, it is. Since many tasks are associated with a single module, there's no deterministic to say "You were associated with this module, so we're going to associate you with this one task."

Right now it's a moot point since there are no evaluation answers